src/block/block_addr.c

```diff
@@ -190,9 +186,8 @@ __wt_block_buffer_to_ckpt(WT_SESSION_IMPL *session,
  * version.
  */
 int
-__wt_block_ckpt_decode(WT_SESSION *wt_session,
-    size_t allocsize, const uint8_t *p, WT_BLOCK_CKPT *ci)
-    WT_GCC_FUNC_ATTRIBUTE((visibility("default")))
+__wt_block_ckpt_decode(WT_SESSION *wt_session, size_t allocsize, const uint8_t *p,
+    WT_BLOCK_CKPT *ci) WT_GCC_FUNC_ATTRIBUTE((visibility("default")))
```

Comment:
More issues with macros. Ideally it'd be on its own line but the formatter removes the line break.